bin/sirius-script-ps-devices.py

```diff
@@ -174,5 +191,10 @@ def create_tree():
     return tree
 
 
-tree = create_tree()
-print(format_tree_simple(tree))
+if __name__ == "__main__":
+    tree = create_tree()
+    if len(sys.argv) == 1:
```

Comment:
I don't know if you're going to add more options, but if you are, you could pass the options to a variable and use the print and the format_tree_simple only once at the end, then it would be easier to add more options.